src/index.coffee

```diff
@@ -91,7 +91,12 @@ module.exports = (projectConfig={}) ->
     runSequence('clean', ['src', 'static', 'style'])
 
 
-  gulp.task 'watch:serve', ['watch', 'serve']
+  gulp.task 'serve', ->
+    runSequence('clean', ['src', 'static', 'style'], ['_serve'])
```

Comment:
I tried that, but runSequence overall returns asynchronously.

On January 29, 2016 12:10:58 PM EST, Casey Liscum <notifications@github.com> wrote:
>> @@ -91,7 +91,12 @@ module.exports = (projectConfig={}) ->
>>      runSequence('clean', ['src', 'static', 'style'])
>>  
>>  
>> -  gulp.task 'watch:serve', ['watch', 'serve']
>> +  gulp.task 'serve', ->
>> +    runSequence('clean', ['src', 'static', 'style'], ['_serve'])
>
>Can this be `runSequence('build', '_serve')`?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/vistarmedia/gulp-tasks/pull/12/files#r51287004